packages/core/core/src/Parcel.js

```diff
@@ -42,33 +50,12 @@ export default class Parcel {
       this.options.env = process.env;
     }
 
-    let configResolver = new ConfigResolver();
```

Comment:
Didn't totally think through moving this code out. We will still need to resolve a config at the root to get the right bundler/packager/reporter etc. plugins.